src/main/webapp/js/stats-page-loader.js

```diff
@@ -0,0 +1,24 @@
+// fetch stats and display them in page
+function fetchStats() {
+    const url = '/stats';
+    fetch(url).then((response) => {
+        return response.json();
+    }).then((stats) => {
+        const statsContainer = document.getElementById('stats-container');
+        statsContainer.innerHTML = '';
+
+        const messageCountElement = buildStatElement('Message count: ' + stats.messageCount);
+        statsContainer.appendChild(messageCountElement);
+    });
+}
+
+function buildStatElement(statString) {
+    const statElement = document.createElement('p');
```

Comment:
Is 'p' here arbitrary or was it taken from another file?

Reply:
I believe it's arbitrary, but again, I was following the tutorial.